vlasovsolver/cpu_moments.cpp

```diff
@@ -147,9 +144,12 @@ void calculateCellMoments(spatial_cell::SpatialCell* cell,
       pop.P[2] = mass*array[2];
```

Comment:
For multipop shenanigans later on; I would recommend storing the full pressure tensor for all populations and refining if any population has a pressure anisotropy which would trigger the threshold.

Reply:
If we want it later on, I think we should implement it later on

Reply:
> To get our goals on the road, this can be postponed for a bit, but we cannot have code breaking multipop behaviour just because we aren't running multipop actively at this time

Unless I'm missing something it's not breaking multipop behaviour? It's just saving the off-diagonal bulk pressure to the CellParams, where we used to store the diagonal bulk pressure already. I can add the pop-wise elements to `Population.P` but chose not to for this PR because:
- It wasn't needed for the feature coded
- I genuinely have no idea how entangled that bit is. e.g. do we loop through _all_ (not three) elements of pressure somewhere?

Reply:
Break not in the sense of crashing the code, but in the sense that it will not refine when just a single population fulfills the criterion, while bulk pressure anisotropy not. And that may then lead to loss of beams for a less-common population.

So at the point of this comment, add pop.P[3...5] additions, and manage them as well when evaluating pressure anisotropy. And you'll want to add per-population pressure anisotropy data reducers as well.